app/services/remove_status_service.rb

```diff
@@ -103,6 +109,8 @@ def remove_reblogs
     # without us being able to do all the fancy stuff
 
     @status.reblogs.rewhere(deleted_at: [nil, @status.deleted_at]).includes(:account).reorder(nil).find_each do |reblog|
+      # We don't pass through deliver_quietly? here as the Reblog may be much
+      # newer than the status being removed
       RemoveStatusService.new.call(reblog, original_removed: true)
```

Comment:
I think you should pass it. While reblogs may be much newer, it's supposed to have been deleted 30 days ago, like the original post being deleted.

Reply:
Apparently I didn't push the commit that resolves this, will fix later today!